src/app/loadout-builder/types.ts

```diff
@@ -65,15 +66,19 @@ export type LockedMap = Readonly<{
   [bucketHash: number]: readonly LockedItemType[] | undefined;
 }>;
 
+export const ModPickerCategories = { ...Armor2ModPlugCategories, seasonal: 'seasonal' } as const;
+export type ModPickerCategory = typeof ModPickerCategories[keyof typeof ModPickerCategories];
+
 export interface LockedArmor2Mod {
   mod: DestinyInventoryItemDefinition;
-  category: number | 'seasonal';
+  category: ModPickerCategory;
 }
 
-export type LockedArmor2ModMap = Readonly<{
-  [categoryHash: number]: readonly LockedArmor2Mod[] | undefined;
-  seasonal?: readonly LockedArmor2Mod[];
-}>;
+export type LockedArmor2ModMap = Readonly<
```

Comment:
This is fine, though it's equivalent to:

```
{
  [T in ModPickerCategory]: readonly LockedArmor2Mod[];
}
```

Reply:
Done